### ISSUE TITLE
can set # of required dungeons/medallions/stones/etc. to 0

### ISSUE THREAD

**Issue by briaguya-ai** [briaguya-ai/rando-issue-tracker#90]
this is an option in the 3ds logic so i don't know if it's a problem really, but it does seem like a roundabout way of setting "open bridge"

figured it's worth noting

![image](https://user-images.githubusercontent.com/70942617/177202476-1fb8cbb5-8037-492e-97c7-a3fbceab4c1a.png)

**Comment by briaguya-ai** [comment]
the reason i made this a "maybe" is because it'll involve testing every option to ensure we correctly set the index of the setting in 3drando and the spoiler is generated correctly and the bridge works as expected. also allowing 0 doesn't break anything.

**Comment by aMannus** [comment]
Yeah currently if we just remove 0, it'll shift every other option by 1 because it's all index based right now. AKA choosing "1 medallion required" will actually make the logic use "0 medallions required" unless we change the 3DS logic too.

**Comment by briaguya-ai** [comment]
Yeah, we'd need to just throw a +1 somewhere in there, so I'd expect the code change to be tiny, but would still want to test heavily
